scripts/kubernetes/serversChart/nginx/nginx.conf

```diff
@@ -0,0 +1,21 @@
+server {
```

Comment:
can we leverage the one in sample-web-ui?

Reply:
was having difficulties using a relative path to the config during deployment plus i was thinking it creates a dependency on that repo. does the config change often?

Reply:
I think the dependency is fine

Reply:
https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit/issues/35